scripts/turtles-dev.sh

```diff
@@ -48,10 +48,10 @@ export CLUSTER_TOPOLOGY=true
 
 helm install capi-operator capi-operator/cluster-api-operator \
     --create-namespace -n capi-operator-system \
-    --set infrastructure=docker:v1.7.7 \
-    --set core=cluster-api:v1.7.7 \
-    --set controlPlane=rke2:v0.8.0 \
-    --set bootstrap=rke2:v0.8.0 \
+    --set infrastructure=docker \
```

Comment:
the only objective I would have is: with this change, we start installing CAPI v1.8.5 and CAPRKE2 v0.8.0 where latter is compatible only with CAPI v1.7.x series